File: 0001 Code Chef/0001 Advitiya/implementation/0009 dart/advitiya.dart
import 'dart:io';


void main(){
    String? stringInput = stdin.readLineSync();
    int ? day = int.tryParse(stringInput!);
    if(day != null){
        if(day == 17 || day == 18 || day == 19){
          print("ADVITIYA");
        }else{
          print("WAITING FOR ADVITIYA");
        }
    }else{
      print("Input is null");
    }
}

File: 0001 Code Chef/0001 Advitiya/implementation/0010 nodejs/advitiya.js
const readline = require('readline');


const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});


rl.question('',(input)=>{
    let day = parseInt(input);
    if(day == 16 || day == 17 || day == 18){
        console.log("ADVITIYA");
    }else{
        console.log("WAITING FOR ADVITIYA");
    }
    rl.close();
})